apps/shipments/views.py

```diff
@@ -142,17 +143,18 @@ class LocationViewSet(viewsets.ModelViewSet):
     queryset = Location.objects.all()
     serializer_class = LocationSerializer
     # TODO: Clarify/Solidify the permissions for Locations w/ respect to owner_id
-    permission_classes = (permissions.IsAuthenticated, IsOwner) if settings.PROFILES_URL else (permissions.AllowAny,)
+    permission_classes = (permissions.IsAuthenticated, IsOwner) if settings.PROFILES_URL != 'DISABLED'\
+        else (permissions.AllowAny,)
     http_method_names = ['get', 'post', 'delete', 'patch']
 
     def get_queryset(self):
         queryset = self.queryset
-        if settings.PROFILES_URL:
+        if settings.PROFILES_URL != 'DISABLED':
             queryset = queryset.filter(owner_id=self.request.user.id)
         return queryset
 
     def perform_create(self, serializer):
-        if settings.PROFILES_URL:
+        if settings.PROFILES_URL != 'DISABLED':
```

Comment:
Instead of adding this boolean check in profiles, maybe add a setting like `PROFILES_ENABLED = PROFILES_URL != 'DISABLED'` and then you can just reference this like `if settings.PROFILES_ENABLED`.